sdks/java/core/src/main/java/org/apache/beam/sdk/transforms/display/DisplayData.java

```diff
@@ -306,9 +308,27 @@ private static Item create(ItemSpec<?> spec, Path path) {
           spec.getValue(), spec.getShortValue(), spec.getLabel(), spec.getLinkUrl());
     }
 
+    /** <b>For internal use only. No backwards compatibility guarantees.</b> */
+    @Internal
+    public static Item raw(
```

Comment:
The unit tests currently go through the standard Builder interface to build an ItemSpec and then let standard internal mechanisms translate that into DisplayData.Items.

Having a single way of building Items makes it easier to reason about the construction logic. I'm not opposed to adding a second if necessary, but we should make sure it has the same validation and translation for edge-cases.

ProxyInvocationHandler has similar needs to construct DisplayData from deserialized JSON. Check out `ProxyInvocationHandler.DisplayDataValue#resolve()` and `DisplayData.inferType()`.